src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -124,9 +124,6 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $io = new SymfonyStyle($input, $output);
-        $errorIo = $output instanceof ConsoleOutputInterface ? new SymfonyStyle($input, $output->getErrorOutput()) : $io;
```

Comment:
Check some lines below, `$io` is initialized again

Reply:
Won't the command break at l128? I guess the second io initialization below is the one that should be removed instead

Reply:
You could be right, not tested

Reply:
I think we should move the lower one up